### PR TITLE
Feature/converter/hint for service tasks

### LINKED ISSUE #84: Improve hint for service tasks
### Is your feature request related to a problem? Please describe.

Currently, the new value is not mentioned:

```
TASK: Attribute 'class' on 'serviceTask' was mapped, please review. Hint: Delegate call to 'com.camunda.consulting.CreateTweetDelegate' was transformed to job type. Please review your implementation
```

### Describe the solution you'd like

The hint should look like:

```
Delegate call to 'com.camunda.consulting.CreateTweetDelegate' was transformed to job type 'camunda-7-adapter'. Please review your implementation.
```

### Describe alternatives you've considered

Alternativlos

### Additional context

nothing

### PR DESCRIPTION
**Pull request by ingorichtsmeier**
# Improve hint for service tasks

## Description

Mention the task type in the message for the conversion.

## Additional context

Fixes #84 

## Testing your changes

See `org.camunda.community.converter.BpmnConverterTest.testDelegateHint()`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [x] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [ ] Documentation update (changes made to an existing piece of documentation)

## Checklist:

- [x] My code adheres to the syntax used by this extension.
- [ ] My pull request requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **Camunda Community Hub** documentation.
- [ ] I have read the **Pull Request Process** documentation.
- [x] I have added or suggested tests to cover my changes suggested in this pull request.
- [ ] All new and existing CI/CD tests passed.
- [ ] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [ ] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

**Comment by jonathanlukas** [comment]
This one will break on merging #91 

I would suggest to rebase after #91 is merged and see if the problem persists.

File: backend-diagram-converter/core/src/main/java/org/camunda/community/converter/BpmnDiagramCheckResult.java [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
 public class BpmnDiagramCheckResult {
-  private String filename;
   private final List<BpmnElementCheckResult> results = new ArrayList<>();
+  private String filename;
 
   public List<BpmnElementCheckResult> getResults() {
     return results;
   }
 
+  public BpmnElementCheckResult getResult(String elementId) {
```
Comment (jonathanlukas) [inline backend-diagram-converter/core/src/main/java/org/camunda/community/converter/BpmnDiagramCheckResult.java:14]: Do we need this method here? I would try to keep all Pojos as clean as possible (only simple getter and setter).

Maybe we could extract it to a helper class.

**Review by jonathanlukas — CHANGES_REQUESTED** [review]

File: backend-diagram-converter/core/src/main/java/org/camunda/community/converter/BpmnDiagramCheckResult.java (hunk 1 above)
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/java/org/camunda/community/converter/BpmnDiagramCheckResult.java:14]: I once had this as a map. Then, I decided that it should be a list. The reason was back then that iterating a list is way easier in js as iterating a map.

So, we can change it which will cause some refactoring.

If you agree, I would open a ticket for this and we leave the method as it is in the meantime.

Comment (ingorichtsmeier) [reply backend-diagram-converter/core/src/main/java/org/camunda/community/converter/BpmnDiagramCheckResult.java:14]: We should keep both methods `List<BpmnElementCheckResult> getResults()` and `BpmnElementCheckResult getResult(String elementId)`.

Refactoring can be done in this ticket as only the internals change and the tests are stable.

**Review by jonathanlukas — APPROVED** [review]

**Pull request merged**